ClosedXML/Excel/InsertData/InsertDataReaderFactory.cs

```diff
@@ -13,12 +13,15 @@ internal class InsertDataReaderFactory
 
         public static InsertDataReaderFactory Instance => _instance.Value;
 
-        public IInsertDataReader CreateReader<T>(IEnumerable<T> data)
+        public IInsertDataReader CreateReader(IEnumerable data)
```

Comment:
Was it not possible to keep both the generic and non-generic as 2 overloads?

Reply:
This would not add any value but would bring more confusion during type casting. Like, I've just found that `IEnumerable<string>` produces `ArrayReader` as the argument is treated as `IEnumerable<IEnumerable>`. Gonna fix this.